models/ios/lacp_interfaces/ios_lacp_interfaces.yaml

```diff
@@ -0,0 +1,53 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+NETWORK_OS: ios
+RESOURCE: lacp_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: ios_lacp_interfaces
+  version_added: 2.9
+  short_description: Manage Link Aggregation Control Protocol (LACP) on Cisco IOS devices interface.
+  description: This module provides declarative management of LACP on Cisco IOS network devices interfaces.
+  author: Sumit Jaiswal (@justjais)
+  notes:
+  - Tested against Cisco IOSv Version 15.2 on VIRL
+  options:
+    config:
```

Comment:
There's probably also a scope of adding `fast-switchover` and `max-bundle` lacp options for port-channels.